HashTable.java

```diff
@@ -0,0 +1,192 @@
+import java.util.*;
+import java.lang.reflect.Array;
+
+class HashTable <K, V> {
```

Comment:
Add a comment here in the following form:

/**
 * Simple sentence description of what this class does.
 * 
 * If needed, more paragraphs describing what the class does, why it exists and how it fits in.
 * 
 * @author Louis Ashton (louisashton@live.com)
 */